firstvoices/backend/search/query_builder.py

```diff
@@ -86,16 +86,16 @@ def get_search_query(
             )
 
     # Add site filter if parameter provided in url
-    if site_id:
-        search_query = search_query.query(get_site_filter_query(site_id))
+    if sites:
+        search_query = search_query.query(get_site_filter_query(sites))
 
     types_query = get_types_query(types)
     if types_query:
         search_query = search_query.query(get_types_query(types))
 
     if starts_with_char:
         search_query = search_query.query(
-            get_starts_with_query(site_id, starts_with_char)
+            get_starts_with_query(sites[0], starts_with_char)
         )
```

Comment:
Oh interesting combo case.... I'm going to log a ticket to design how this should actually work. This will work for now but it will give unexpected results if people try to use starts_with_char in the cross-site search